[Edit] Small edit to fix syntax errors in rdf.md (#3945)
Codecademy/docs: content/general/concepts/semantic-technologies/terms/rdf/rdf.md

- _IRI_: A generalization of [URI](https://www.codecademy.com/resources/docs/general/uri), IRIs identify and link to resources on private and public networks.
- _Quoted triple_: An RDF triple used within a different RDF triple. Quoted triples allow statements to be made within RDF about other RDF statements.
- _Literal_: A value representing a number, string, boolean, date/time, or other simple values.

> - _Blank node_: Blank nodes do not specify anything in particular. They allow for statements that say that something stands in a given relationship without identifying what that thing is.

Reviewer: ```suggestion
- _Blank node_: Does not specify anything in particular. Allows statements stating that something stands in a given relationship without identifying what that thing is.
```
